eva: add page (#3490)
tldr-pages/tldr: pages/common/eva.md

# eva

> Simple calculator REPL, similar to `bc`, with syntax highlighting and persistent history.
> More information <https://github.com/NerdyPepper/eva>.

> - Run calculator in interactive mode:

Reviewer: ```suggestion
- Run the calculator in interactive mode:
```

`eva`

- Calculate the result of an expression:

> `eva <<< "{{(1 + 2) * 2 ^ 2}}"`

Reviewer: I see from the linked homepage (https://github.com/NerdyPepper/eva) that this could be done by simply supplying the expression as first argument. Like:

```suggestion
`eva "{{(1 + 2) * 2 ^ 2}}"`
```

This and the following examples should be modified accordingly, there is no need to use the bash redirection `<<<`.

> - Calculate expression and force number of decimal places to 5:

Reviewer: ```suggestion
- Calculate an expression forcing the number of decimal places to 5:
```

`eva --fix {{5}} <<< "{{5 / 3}}"`

> - Calculate expression with sine and cosine:

Reviewer: ```suggestion
- Calculate an expression with sine and cosine:
```
